src/views/cic/enter-name-hmrc-check.html

```diff
@@ -0,0 +1,121 @@
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "hmpo-text/macro.njk" import hmpoText %}
+{% from "hmpo-form/macro.njk" import hmpoForm %}
+
+{% block mainContent %}
+
+
+{% set detailsBody %}
+<div>
+  <p>{{ translate("nameEntry.details.line1") }}</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li class="govuk-!-margin-left-3">{{ translate("nameEntry.details.bullet1") }}</li>
+      <li class="govuk-!-margin-left-3">{{ translate("nameEntry.details.bullet2") }}</li>
+    </ul>
+    <p>{{ translate("nameEntry.details.line2") }}</p>
+    <p>{{ translate("nameEntry.details.line3") }}</p>
+    <p>{{ translate("nameEntry.details.line4") }}</p>
+    <p>{{ translate("nameEntry.details.line5") }}<a href={{ "nameEntry.contactSupport.link" | translate }} id="contactSupport"> {{ "nameEntry.contactSupport.text" | translate }}</a></p>
+</div>
+{% endset %}
+
+
+  <div>
+    <h1 id="header" class="gov" data-page="{{hmpoPageKey}}" class="govuk-heading-l">
+        {{ title }}
+    </h1>
+
+    <div id="lowConfidenceSubtext">
+      <p>{{ translate("nameEntry.LOW_CONFIDENCE.subtext") }}</p>
+    </div>
+
+
+    {% call hmpoForm(ctx) %}
+
+    {{ hmpoText(ctx, {
+        id: "surname",
+        autocomplete: "family-name",
+        classes: "govuk-!-width-full"
+    })}}
+
+    {%- set firstNameError = hmpoGetError(ctx, {id: 'firstName'}) %}
+    {%- set middleNameError = hmpoGetError(ctx, {id: 'middleName'}) %}
+
+    {% if firstNameError and middleNameError %}
+        <div class="govuk-form-group govuk-form-group--error">
+    {% else %}
+        <div class="govuk-form-group ">
+      {% endif %}
+
+    {% call govukFieldset({
+      legend: {
+        text: translate("nameEntry.givenNames"),
+        classes: "govuk-!-margin-0"
+      }
+    }) %}
+        <div class="govuk-inset-text govuk-!-margin-top-2 govuk-!-padding-top-1">
+            {{ hmpoText(ctx, {
+              id: "firstName",
+              label: {
+              classes: "govuk-label"
+              },
+              classes: "govuk-input",
+              autocomplete: "given-name"
+              }) }}
+
+            {{ hmpoText(ctx, {
+              id: "middleName",
+              label: {
+              classes: "govuk-label"
+              },
+              classes: "govuk-input",
+              autocomplete: "additional-name"
+              }) }}
+        </div>
+      {% endcall %}
+    </div>
+
+
+    {{ hmpoSubmit(ctx, {classes: "govuk-!-margin-top-0", attributes: {"data-testid": "enter-name-continue-btn"}, text: translate("buttons.next")}) }}
+          <div>
+            {{ govukDetails({
+            id: "characterDetails",
+            classes: "govuk-!-margin-bottom-7",
+            summaryText: translate("nameEntry.details.summary"),
+            html: detailsBody
+            }) }}
+          </div>
```

Comment:
This details component should be before the submit button, as per the screens attached to the ticket